backend/cloud_inquisitor/plugins/types/resources.py

```diff
@@ -1502,7 +1502,7 @@ def update(self, data, properties):
         """
 
         updated = self.set_property('metrics', properties['metrics'])
-        updated |= self.set_property('creation_date', data.creation_date)
+        updated |= self.set_property('resource_creation_date', properties['creation_date'])
```

Comment:
`resource_creation_date` is computed from `creation_date`. I think we need to revert this one

Reply:
will fix.